packages/dds/map/src/directory.ts

```diff
@@ -1278,7 +1278,7 @@ class SubDirectory extends TypedEventEmitter<IDirectoryEvents> implements IDirec
                 0x00f /* `pendingMessageId is missing from the local client's ${op.type} operation` */);
```

Comment:
I updated the eslint ignores to ignore the rules in my latest update. @alexvy86 Please let me know if you think that's wrong.

Reply:
Hmm did the `${op.type}` part of the comment come back recently (maybe during a merge)? I thought I cleaned up all the existing ones when I introduced the mapping file.

I'd say instead of disabling `no-template-curly-in-string`, we remove the backticks and the interpolated piece from this comment, so policy-check sees a plain string. The tool should already error out if we try to add new interpolated strings in `assert()` calls, and asks that we replace them with plain strings

And nice change, moving the rules for that map file to `.eslintrc.js` 👍 .

Reply:
Ok yeah, a [rollback of other changes](https://github.com/microsoft/FluidFramework/pull/10615/) in `directory.ts` brought back the old version of the comment. I'll do a PR to clean it up and remove the other eslint rule for the mapping file.